src/ai/BotNames.txt

```diff
@@ -669,7 +667,6 @@
 <r>҉Fiz҉z҉y🚽ᴳᵒᵈ
 <r>๖ۣۜℓuca∞༻一๖大❥❥
 <r>Chinese
-<r>PENIS EATER
```

Comment:
I used a copy-paste names extension then copied every name from like 30 party servers and pasted them into this file, I guess some bad ones made it in there too xD